integration_test/README.md

```diff
@@ -242,6 +242,8 @@ the corresponding `main_log.txt` from there.
 
 Here is the full contents uploaded to the GCS bucket for a single test run.
 The "Details" link takes you directly to the "logs" subdirectory to save you
+a hop. The following is sorted roughly in descending order of usefulness.
```

Comment:
The files below are sorted alphabetically, so maybe not worth mentioning the sort order…
```suggestion
a hop.
```

Reply:
Done in [46db9df](https://github.com/GoogleCloudPlatform/ops-agent/pull/1753/commits/46db9dfee78ae8468c868e5f14699160b2b79c78)